contracts/entry-point/src/contract.rs

```diff
@@ -85,6 +91,48 @@ pub fn execute(
     msg: ExecuteMsg,
 ) -> ContractResult<Response> {
     match msg {
+        ExecuteMsg::AxelarSwapAndAction {
+            user_swap,
+            min_coin,
+            timeout_timestamp,
+            post_swap_action,
+            affiliates,
+            recovery_addr,
+        } => {
+            // Store all parameters into a temporary storage.
+            SWAP_AND_ACTION_REQUEST_TEMP_STORAGE.save(
```

Comment:
I think we only need to store the recovery_addr and funds, right? All other fields seem to be unused

Reply:
ahh good point, just updated!